src/core/bindEvents.js

```diff
@@ -64,13 +69,22 @@ function bindEvents(){
         "."+facet.facetClass, 
         this.findChangedFacet.bind(this)
     )
-    if(facet.selectedFacetsEl) {
+    if(selectedFacets && selectedFacets.el){
```

Comment:
I think this should check only for `selectedFacets` being there, because the current condition would fail if they give el as null but have provided the other selected facet configs like `selectedFacets.facetAction` and `selectedFacets.selectedFacetClass`

Reply:
we dont need other options in this case bcz selectedfacets wont render if they are not providing el.